src/Symfony/Bundle/FrameworkBundle/Test/WebTestCase.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Symfony\Bundle\FrameworkBundle\Test;
 
+use Symfony\Bundle\FrameworkBundle\Client;
```

Comment:
actually the class is deprecated in 4.3, this should reference `KernelBrowser` now

Reply:
Ok. I was ready for this comment :)